src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -555,7 +556,8 @@ bool memberHasBadState(Symbol member, LocalState state)
 
             void makeNotNullMembersMaybeNull()
             {
-                if (_symbol is MethodSymbol method)
+                var method = _symbol as MethodSymbol;
+                if (method is object)
```

Comment:
```suggestion
                if (_symbol is MethodSymbol method)
```

Reply:
That was the syntax before this change. Unfortunately, that does not allow an annotation on `method`. (`if (_symbol is MethodSymbol? method)` is not valid syntax.)

---
In reply to: [463853593](https://github.com/dotnet/roslyn/pull/45640#discussion_r463853593) [](ancestors = 463853593)